### DISCUSSION TITLE
[2726890] 拜托！明天和我恋爱吧 [仅本体]

### DISCUSSION THREAD

**Discussion by segadcss** [pjy612/SteamManifestCache#442] [Share]
`appid:2726890`
keys:

```
2726891;294bda221d4642787797dae2d1d979e24afb2c54fb277a6d9fa920b521200bb7
```

`

[2726891_1654995095082648032.zip](https://github.com/pjy612/SteamManifestCache/files/14110852/2726891_1654995095082648032.zip)

email:segadcss@outlook.com

code:4CA2BADACCA21A3A

**Comment by ZEERDEER** [comment]
* 格式不对 #26

**Comment by pjy612** [comment]
* 密钥的 depotid 贴错了
* 可惜这个似乎有别的验证，不能随便玩

**Reply by segadcss** [reply]
现在可以用离线号过开头就行
账号IdAuZ176526        密码RzHjF379668

**Reply by segadcss** [reply]
现在还可以，官方说要封号，趁现在玩完拉倒

**Reply by pjy612** [reply]
知道。说的是 必须有个号，还全程联网。不像其他的有办法蹭。。。
